ocp-resources/e2e/spo-install.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: openshift-security-profiles
+labels:
+  openshift.io/cluster-monitoring: "true"
```

Comment:
Looks like this is broken in https://github.com/ComplianceAsCode/content/pull/12315#issuecomment-2299754229